cypress/cypress-integration/integration/pending-motions-table-judge-and-chamber-dashboard.cy.ts

```diff
@@ -105,18 +113,20 @@ function generateRandomDate() {
   };
 }
 
-function getRandomElementFromArray() {
-  const statusOfMatter = [
-    'Awaiting Consideration',
-    'Awaiting Briefs',
-    'Drafting',
-    'Reviewing Draft',
-    'Submitted to Chief Judge',
-    'Revising Draft',
-    'Submitted to Reporter',
-    'Awaiting Release',
-    'Stayed',
-  ];
-  const randomIndex = Math.floor(Math.random() * statusOfMatter.length);
-  return statusOfMatter[randomIndex];
+function selectRandomStatusOfMatterEntry(): [string, string] {
+  const statusOfMatter = {
+    AwaitingBriefs: 'Awaiting Briefs',
+    AwaitingConsideration: 'Awaiting Consideration',
+    AwaitingRelease: 'Awaiting Release',
+    Drafting: 'Drafting',
+    ReviewingDraft: 'Reviewing Draft',
+    RevisingDraft: 'Revising Draft',
+    Stayed: 'Stayed',
+    SubmittedToChiefJudge: 'Submitted to Chief Judge',
+    SubmittedToReporter: 'Submitted to Reporter',
+  };
```

Comment:
Can / should this reference the static property of the class itself?

`DocketEntryWorksheet.STATUS_OF_MATTER_OPTIONS_DICTIONARY` 

This way should that list change, we don't have to change it in two places?

Reply:
Sadly if we import the entity into a cypress test that will require to update the configuration to allow entities to be bundled with cypress. Which will increase the size and import unnecessary files just for one dictionary.

Reply:
Refactored test to not use dictionary and instead use the markup to get all available options and randomly select one.